ui/src/routes/incarnations/EditForm.tsx

```diff
@@ -9,14 +9,22 @@ import { IncarnationsForm } from './Form'
 import { Section } from './parts'
 import { Incarnation } from '../../interfaces/incarnations.types'
 
-const toIncarnationInput = (x: Incarnation): IncarnationInput => ({
-  automerge: true,
-  repository: x.incarnationRepository,
-  targetDirectory: x.targetDirectory,
-  templateRepository: x.templateRepository ?? '',
-  templateVersion: x.templateRepositoryVersion ?? '',
-  templateData: Object.entries(x.templateData).map(([key, value]) => ({ key, value }))
-})
+const toIncarnationInput = (x: Incarnation): IncarnationInput => {
+  let templateData = '{}'
+  try {
+    templateData = JSON.stringify(x.templateData, null, 2)
```

Comment:
Just want to check. `JSON.stringify` will only throw when a value contains a circular reference or
a `BigInt` value is encountered. Is the intended outcome here to have `templateData` be `undefined` if `JSON.stringify` is unable to serialize, but not throw?  or do you want to keep the default value or `'{}'`?

Reply:
I think I messed here, 'cause JSON should not have BigInt